@here/harp-mapview/lib/text/TextElement.ts

```diff
@@ -491,7 +491,7 @@ export class TextElement {
      */
     constructor(
         readonly text: string,
-        readonly points: THREE.Vector3[] | THREE.Vector3,
+        public points: THREE.Vector3[] | THREE.Vector3,
```

Comment:
Why? Your are only changing the array elements, not the array instance?!

Reply:
done.